x/tokenomics/keeper/keeper_test.go

```diff
@@ -202,20 +206,25 @@ func (s *TestSuite) TestClaimSettlement_ClaimSettled_ProofRequiredAndProvided()
 	expectedEvent, ok := s.getClaimEvent(events, "poktroll.tokenomics.EventClaimSettled").(*tokenomicstypes.EventClaimSettled)
 	require.True(t, ok)
 	require.True(t, expectedEvent.ProofRequired)
-	require.Equal(t, numComputeUnits, expectedEvent.ComputeUnits)
-
+	require.Equal(t, s.numComputeUnits, expectedEvent.ComputeUnits)
 }
 
 func (s *TestSuite) TestClaimSettlement_Settles_WhenAProofIsNotRequired() {
 	// Retrieve default values
 	t := s.T()
 	ctx := s.ctx
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	sharedParams := s.keepers.SharedKeeper.GetParams(ctx)
 
 	// Create a claim that does not require a proof
 	claim := s.claim
-	numComputeUnits := uint64(tokenomicskeeper.ProofRequiredComputeUnits - 1)
-	claim.RootHash = smstRootWithSum(numComputeUnits)
+
+	// Set the proof parameters to NOT require via probability NOR threshold.
```

Comment:
NOR threshold? Not fully understanding the comment

Reply:
Is this clearer?

```suggestion
	// Set the proof parameters to NOT require via NEITHER probability NOR threshold.
```